apps/nextra/pages/en/build/sdks/ts-sdk/_meta.tsx

```diff
@@ -11,18 +11,25 @@ export default {
     title: "Usage",
   },
   "fetch-data-via-sdk": {
-    title: "Fetch Data via the TypeScript SDK",
+    title: "Fetch Data",
   },
   "building-transactions": {
-    title: "Building and Sending Transactions",
+    title: "Transactions",
   },
-  account: {
-    title: "Creating and Managing Accounts",
+  "simulating-transactions": {
+    title: "Simulating Transactions",
   },
-  "legacy-ts-sdk": {
-    title: "Legacy TypeScript SDK",
+  "ts-examples": {
+    title: "Example Code",
+  },
+  account: {
+    title: "Accounts",
```

Comment:
Let's push Accounts to the top (beneath `Usage`)

Reply:
I disagree with pushing `Accounts` to the top because it's a very reference heavy page. Most people who land on it will just be confused (even though logically accounts being first makes sense, the other pages all show how to do that). I'd prefer not to have users actually access it unless they really need it.

Reply:
I understand the perspective here but it's also one of the first things people do when using the TS SDK and looks quite out of place next to `Surf: TS SDK` and `Examples`. I think it would be better to put it at the top, below `Usage`

Reply:
Fair enough